Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -108,4 +108,44 @@
             }
         }
     }
+
+    @Section(title: "Setting Up Firebase Local Emulator") {
+        @ContentAndMedia {
+            Follow these steps to set up the Firebase Local Emulator to load data locally and not affect production. Please do not skip this step.
+        }
+
+        @Steps {
+            @Step {
+                Install Homebrew, a package manager for macOS, if you haven't already:
+                `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+            }
+            @Step {
+                Install Xcode command line tools:
+                `xcode-select --install`
```

Comment:
Can you make this so that it shows up more like this one, where it's the description / text, and then after the horizontal line, it's the command

![Screenshot 2024-10-01 at 7 13 23 PM](https://github.com/user-attachments/assets/d321fa08-8f19-47af-8e95-ea5b286edbcd)